### ISSUE TITLE
Error running ts files in parallel.

### ISSUE THREAD

**Issue by puhnastik** [angular/protractor#3890]
Seems to be an issue when running type script specs in parallel. 
"protractor": "4.0.9"

I have one .ts spec file
`import { IndexPage } from '../pages/index.po';
describe('starter-app App', function() {
  let page: IndexPage;

  beforeEach(() => {
    page = new IndexPage();
    page.get();
  });
  it('should display message saying app works', () => {
    expect(page.getParagraphText()).toEqual('app works!');
  });
});`
and several js files. 
I can successfully run js files in parallel, but not .ts one (either in one browser or in several browsers). The error is thrown on import module in .ts file. 
The same suite executes without errors if specs are executed one by one.

Please, let me know if you can reproduce?

**Comment by juliemr** [comment]
How are you compiling your ts file down to javascript? Protractor does not compile TS for you. Protractor treats all your js files the same, so there should be no issue here.

Also please use github formatting for code.

**Comment by juliemr** [comment]
Please make sure you're compiling your TS properly, and open up a new issue if you're still having a problem. There's some examples that you may find helpful at https://github.com/angular/protractor-cookbook
